components/lcd_touch/esp_lcd_touch_ft5x06/esp_lcd_touch_ft5x06.c

```diff
@@ -120,28 +120,28 @@ esp_err_t esp_lcd_touch_new_i2c_ft5x06(const esp_lcd_panel_io_handle_t io, const
 
     /* Prepare pin for touch interrupt */
     if (esp_lcd_touch_ft5x06->config.int_gpio_num != GPIO_NUM_NC) {
-        const gpio_config_t int_gpio_config = {
+        const gpio_config_t rst_gpio_config = {
             .mode = GPIO_MODE_INPUT,
+            .intr_type = GPIO_INTR_NEGEDGE,
             .pin_bit_mask = BIT64(esp_lcd_touch_ft5x06->config.int_gpio_num)
         };
-        ret = gpio_config(&int_gpio_config);
+        ret = gpio_config(&rst_gpio_config);
         ESP_GOTO_ON_ERROR(ret, err, TAG, "GPIO config failed");
+
+        /* Register interrupt callback */
+        if (esp_lcd_touch_ft5x06->config.interrupt_callback) {
+            esp_lcd_touch_register_interrupt_callback(esp_lcd_touch_ft5x06, esp_lcd_touch_ft5x06->config.interrupt_callback);
+        }
     }
 
     /* Prepare pin for touch controller reset */
     if (esp_lcd_touch_ft5x06->config.rst_gpio_num != GPIO_NUM_NC) {
-        const gpio_config_t rst_gpio_config = {
+        const gpio_config_t int_gpio_config = {
```

Comment:
The name of variable was right.
```suggestion
        const gpio_config_t rst_gpio_config = {
```